acceptance/pre_suite/subcommands/05_install_ruby.rb

```diff
@@ -0,0 +1,32 @@
+test_name 'Install and configure Ruby 2.2.5 on the SUT' do
+
+  step 'Ensure that the default system is an el-based system' do
+    # The pre-suite currently only supports el systems, and we should
+    #fail early if the default platform is not a supported platform
+    assert(default.platform.variant == 'el',
+           "Expected the platform variant to be 'el', not #{default.platform.variant}")
```

Comment:
This one is to align with the arguments for `assert`; I usually do this to so single lines don't exceed 120 chars.